modules/cli/src/main/scala/scala/cli/commands/ScalaCommand.scala

```diff
@@ -159,6 +163,74 @@ abstract class ScalaCommand[T](implicit myParser: Parser[T], help: Help[T])
       sys.exit(exitCode)
     }
 
+  def maybePrintToolsHelp(options: T, buildOptions: BuildOptions): Unit =
+    for {
+      shared <- sharedOptions(options)
+      logger = shared.logger
+      artifacts      <- buildOptions.artifacts(logger, Scope.Main).toOption
+      scalaArtifacts <- artifacts.scalaOpt
+      scalaParams = scalaArtifacts.params
+      if shared.helpGroups.helpScaladoc || shared.helpGroups.helpRepl || shared.helpGroups.helpScalafmt
+    } {
+      val exitCode: Either[BuildException, Int] = either {
+        val (classPath: Seq[os.Path], mainClass: String) =
+          if (shared.helpGroups.helpScaladoc) {
+            val docArtifacts = value {
+              Artifacts.fetch(
+                Positioned.none(Seq(dep"org.scala-lang::scaladoc:${scalaParams.scalaVersion}")),
+                buildOptions.finalRepositories,
+                Some(scalaParams),
+                logger,
+                buildOptions.finalCache,
+                None
+              )
+            }
+            (docArtifacts.files.map(os.Path(_, os.pwd)), "dotty.tools.scaladoc.Main")
```

Comment:
NIT I prefer the `->` syntax if there's a lot of brackets for improved readability
```suggestion
            docArtifacts.files.map(os.Path(_, os.pwd)) -> "dotty.tools.scaladoc.Main"
```